app/views/declaration/summary/items_section.scala.html

```diff
@@ -14,33 +14,19 @@
  * limitations under the License.
  *@
 
-@import models.declaration.Items
+@import models.requests.JourneyRequest
 @import models.viewmodels.HtmlTableRow
-@import models.DeclarationType.DeclarationType
 
-@(items: Option[Items], declarationType: DeclarationType)(implicit messages: Messages)
+@(declarationData: ExportsDeclaration)(implicit messages: Messages, journeyRequest: JourneyRequest[AnyContent])
```

Comment:
You can get `declarationData`  from the request cant you?

Reply:
Good point, it was previous implementation. I'll remove it right now

Reply:
For every section we will stay with declarationData as argument. We will use implicit one on the `summaryPage`